doc-content/enterprise-only/openshift/ldap_rhsso.adoc

```diff
@@ -92,4 +92,9 @@ endif::centralonly[]
 *** *RH-SSO Disable SSL Certificate Validation* (`SSO_DISABLE_SSL_CERTIFICATE_VALIDATION`): Set to `true` if your RH-SSO installation does not use a valid HTTPS certificate. 
 .. To configure LDAP, set the `AUTH_LDAP*` parameters of the template. These parameters correspond to the settings of the LdatExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/login_module_reference/#ldapextended_login_module[LdapExtended Login Module]. 
 +
+If the LDAP server does not define all the roles required for your deployment, you can map LDAP groups to {PRODUCT} roles. To enable LDAP role mapping, set the following parameters:
++
+*** *RoleMapping rolesProperties file path* (`AUTH_ROLE_MAPPER_ROLES_PROPERTIES`): The fully qualified pathname of a file that defines role mapping, for example, `/opt/eap/standalone/data/rolemapping.properties`. You must provide this file and mount it at this path in all applicable deployment configurations; for instructions, see <<rolemapping-proc>>.
+*** *RoleMapping replaceRole property* (`AUTH_ROLE_MAPPER_REPLACE_ROLE`): If set to `true`, mapped roles replace the roles defined on the LDAP server; if set to `false`, both mapped roles and roles defined on the LDAP server are set as user application roles. The default setting is `false`.
++
 Do not configure LDAP authentication and RH-SSO authentication in the same deployment.
```

Comment:
This statement if applicable to both the bullets, needs to be indented left in the same level as point c.